src/components/layouts/Groundfloor.vue

```diff
@@ -2,12 +2,12 @@
     <div class="groundfloor flex-row">
 
         <div>
-            <transition name="fade">
+            <transition name="fade" v-if="$route.path !== '/network'">
```

Comment:
This solution might not scale well if we add new pages that do not use groundfloor

Reply:
We'll cross that bridge when we get to it. It's unlikely that there will be a new page equivalent to the network. The current way of having the network "behind" the other pages, as to visualize that the network is the foundation of the app (going to the network page is like peeling away the top-layer).